recipes/bowtie/build.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+if [ "$(uname -m)" == "arm64" ]; then
+    make POPCNT_CAPABILITY=0
```

Comment:
Maybe a brief comment on why POPCNT_CAPABILITY needs to be disabled for arm64 only would be great!

Reply:
@corneliusroemer 
I've added a comment about it. Thank you!

Reply:
I see that debian has the same patch: https://sources.debian.org/patches/bowtie/1.3.0%2Bdfsg1-1/popcnt_capability.patch/ :)